scrapy_poet/backend.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+import attr
+import scrapy
+from scrapy.utils.defer import maybe_deferred_to_future
+from web_poet import HttpRequest
+from web_poet.exceptions import (
+    HttpError,
+    HttpRequestError,
+    RequestBackendError,
+)
+
+from scrapy_poet.utils import scrapy_response_to_http_response
+
+logger = logging.getLogger(__name__)
+
+
+def create_scrapy_backend(backend):
+    async def scrapy_backend(request: HttpRequest):
+        if not isinstance(request, HttpRequest):
+            raise TypeError(
+                f"The request should be 'web_poet.HttpRequest' but received "
+                f"one of type: '{type(request)}'."
+            )
+
+        try:
+            request = scrapy.Request(**attr.asdict(request), dont_filter=True)
```

Comment:
I think it'd be good to create an explicit function to convert from web_poet.HttpRequest to Scrapy's Request, and add tests for it - check that headers work, etc. 
``attr.asdict(request)`` looks a bit risky; it'd also break after we introduce URL classes.